lib/webpacker/dev_server_proxy.rb

```diff
@@ -11,6 +11,12 @@ def rewrite_response(response)
   def perform_request(env)
     if env["PATH_INFO"] =~ /#{public_output_uri_path}/ && Webpacker.dev_server.running?
       env["HTTP_HOST"] = Webpacker.dev_server.host_with_port
+
+      if env["HTTP_X_FORWARDED_HOST"]
+        env["HTTP_X_FORWARDED_HOST"] = Webpacker.dev_server.host_with_port
+        env["HTTP_X_FORWARDED_SERVER"] = Webpacker.dev_server.host_with_port
+      end
```

Comment:
Does it matter if an existing `env["HTTP_X_FORWARDED_HOST"]` is present or could we just set these all regardless? e.g: 
```ruby
env["HTTP_HOST"] = env["HTTP_X_FORWARDED_HOST"] = env["HTTP_X_FORWARDED_SERVER"] = Webpacker.dev_server.host_with_port`
```

Reply:
Don't think it matters since it's all going to be forwarded to dev server anyway in development. Will push an update 👍

Reply:
Cool. :shipit: